internal/nix/input_test.go

```diff
@@ -117,17 +118,24 @@ func (l *lockfile) ProjectDir() string {
 	return l.projectDir
 }
 
-func (lockfile) Resolve(pkg string) (string, error) {
+func (l *lockfile) IsVersionedPackage(pkg string) bool {
```

Comment:
Is it necessary for this given package to be in the lockfile?

Reply:
Good call. I can't put it in searcher because of a dependency cycle, but I can remove the receiver so it no longer needs to be mocked.